src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$width-card: 202px;
```

Comment:
why not 200?

Reply:
Because she added borders. It's easier for me to calculate the total width this way)

Reply:
I didn't change it

Reply:
I changed it